models/attribution.test.js

```diff
@@ -24,6 +24,7 @@ describe('attribution', () => {
     fileInfo: {
       rawUrl: 'https://commons.wikimedia.org/wiki/File:Eisklettern_kl_engstligenfall.jpg',
       title: 'File:Eisklettern kl engstligenfall.jpg',
```

Comment:
Is the `title` still used somewhere or can it be removed completetely?

Reply:
Nevermind, looks like `routes/fileinfo.js` uses the `title` attribute for requesting the license.

Reply:
Looks like it can be removed to me.